gam_app/management/commands/ImagesToItems.py

```diff
@@ -1,14 +1,14 @@
 from django.core.management.base import BaseCommand, CommandError
-from models import *
+from gam_app. models import *
```

Comment:
Style: remove the space so that it's `gam_app.models`.

Try to avoid star imports in general because you can't see what's being imported. In this case you only need a couple of names (I think just `Imagen` and `Null`) so it's better to list those out explicitly.